cli/integration_tests/basic_monorepo/dry_run.t

```diff
@@ -19,37 +19,46 @@ Setup
-    Command                 = echo 'building'                                                                                        
-    Outputs                 =                                                                                                        
-    Log File                = packages/util/.turbo/turbo-build.log                                                                   
-    Dependencies            =                                                                                                        
-    Dependendents           =                                                                                                        
-    Inputs Files Considered = 1                                                                                                      
-    ResolvedTaskDefinition  = {"outputs":[],"cache":true,"dependsOn":[],"inputs":[],"outputMode":"full","env":[],"persistent":false} 
+    Task                             = build                                                                                                            
+    Package                          = util                                                                                                             
+    Hash                             = 1a3651e1149bfaf7                                                                                                 
+    Cached (Local)                   = false                                                                                                            
+    Cached (Remote)                  = false                                                                                                            
+    Directory                        = packages/util                                                                                                    
+    Command                          = echo 'building'                                                                                                  
+    Outputs                          =                                                                                                                  
+    Log File                         = packages/util/.turbo/turbo-build.log                                                                             
+    Dependencies                     =                                                                                                                  
+    Dependendents                    =                                                                                                                  
+    Inputs Files Considered          = 1                                                                                                                
+    Configured Environment Variables = NODE_ENV=e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855                                        
```

Comment:
In this test case, `NODE_ENV` is _not_ set. But it's included here, because the task hash input includes the string `NODE_ENV=""` . I'm not sure if this was intentional when it was done originally. 

I decided to keep it here, but it could be confusing to read. It might say to the reader: "The `NODE_ENV` env var is set". But what it's really saying is "`NODE_ENV` is part of the task hash input".